array-map/map.js

```diff
@@ -0,0 +1,52 @@
+const arrayNumbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+const arrayLanguages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+function map(array, transform) {
```

Comment:
Here `transform` should be a callback function. Inside the `map` function you should loop over the array and call the `transform` callback function for each item in the array.